docs/user/api/_index.md

```diff
@@ -158,10 +158,10 @@ Response on success:
 
 ## Agent API
 
-### List current running instances
+### List current running instances of metrics subsystem
 
 ```
-GET /agent/api/v1/instances
+GET /agent/api/v1/metrics/instances
```

Comment:
Can we add a note saying a deprecated alias is `/agent/api/v1/instances`?

Reply:
Done, could you check now?